appveyor.yml

```diff
@@ -28,10 +28,6 @@ environment:
   - JULIAVERSION: "julialang/bin/winnt/x64/0.4/julia-0.4-latest-win64.exe"
     PYTHONDIR: "C:\\Python35-x64"
 
-branches:
-  only:
-    - clean-julia-0.4
```

Comment:
should probably change this to only master, otherwise you get duplicate builds (branch and pr)